clang-tools-extra/test/clang-tidy/checkers/modernize/use-using.cpp

```diff
@@ -314,3 +314,10 @@ typedef struct { struct { int a; struct { struct { int b; } c; int d; } e; } f;
 typedef struct { struct { int a; } b; union { int c; float d; struct { int e; }; }; struct { double f; } g; } PR50990_siblings;
 // CHECK-MESSAGES: :[[@LINE-1]]:1: warning: use 'using' instead of 'typedef'
 // CHECK-FIXES: using PR50990_siblings = struct { struct { int a; } b; union { int c; float d; struct { int e; }; }; struct { double f; } g; };
+
+typedef void (*ISSUE_65055_1)(int);
```

Comment:
Original issue were mainly with single character variable names, maybe test something like:
`typedef void (*C)(int);`

Reply:
length of variable names is not related with this bugs.

Reply:
ok